fs/dafsclient/rmtfile.cpp

```diff
@@ -1189,7 +1190,7 @@ class CRemoteFile : public CRemoteBase, implements IFile
     }
 };
 
-class CRemoteFileIO : implements IFileIO, public CInterface
+class CRemoteFileIO : public CInterfaceOf<IFileIO>
```

Comment:
This change is unconnected, but should cause no issues.  It will reduce memory consumption, and the number of thunks for the virtual calls.